src/tools/crossgen2/Common/Compiler/DependencyAnalysis/ArrayOfEmbeddedPointersNode.cs

```diff
@@ -29,7 +35,7 @@ public ArrayOfEmbeddedPointersNode(string startSymbolMangledName, string endSymb
             : base(
                   startSymbolMangledName,
                   endSymbolMangledName,
-                  nodeSorter != null ? new PointerIndirectionNodeComparer(nodeSorter) : null)
```

Comment:
This file (all files under `crossgen2/Common`) is shared with the full AOT compiler. NodeSorter being null is a valid use case there.

Reply:
Ok, reverted.